ruby_on_rails/README.md

```diff
@@ -32,6 +32,7 @@ Most are not optional, but the rest of the team can start working even if those
 3. [robots.txt](robots_txt.md)
 4. [Percy](configure_percy.md) (optional)
 5. [Papertrail](papertrail.md)
+6. [Protect staging environments](environment_protection.md)
```

Comment:
"staging" is not really a word we commonly use. What about `Protect develop and testing environments`